jazz-ui/src/assets/data/jazz-installer-vars.json

```diff
@@ -0,0 +1,97 @@
+{
+  "AWS_CREDENTIAL_ID": "awscreds1",
+  "INSTANCE_PREFIX": "",
+  "AWS": {
+    "SECURITY_GROUP_IDS": "",
+    "SUBNET_IDS": "",
+    "LAMBDA_EXECUTION_ROLE": "",
+    "CLOUDFRONT_ORIGIN_ID": "",
+    "ACCOUNTID": "",
+    "REGION": "",
+    "ROLEID": "",
+    "ES_HOSTNAME": "",
+    "COGNITO": {
+      "USER_POOL_ID": "",
+      "CLIENT_ID": ""
+    },
+    "API": {
+      "PROD": {
+        "*": "{AWS_PROD_API_ID_DEFAULT}",
+        "jazz_*": "{AWS_PROD_API_ID_JAZZ}"
+      },
+      "STG": {
+        "*": "{AWS_STG_API_ID_DEFAULT}",
+        "jazz_*": "{AWS_STG_API_ID_JAZZ}"
+      },
+      "DEV": {
+        "*": "{AWS_DEV_API_ID_DEFAULT}"
+      }
+    },
+    "S3": {
+      "API_DOC": "",
+      "DEV_BUCKET": "",
+      "STG_BUCKET": "",
+      "PROD_BUCKET": ""
+    }
+  },
+  "REPOSITORY": {
+    "BASE_URL": "",
+    "CREDENTIAL_ID": "jenkins1cred",
+    "REPO_BASE_SERVICES": "cas",
+    "REPO_BASE_PLATFORM" : "slf"
+  },
+  "API": {
+    "API_BUILD_URI": "/job/build_pack_api/buildWithParameters?token=slf-java-api-build-101"
```

Comment:
Are these values suppose to be exposed

Reply:
The jazz-installer-vars.json file was taken from the jazz-installer oss repo, i have changed some variables just in case.